docs/pages/versions/unversioned/sdk/video.mdx

```diff
@@ -139,6 +139,100 @@ const styles = StyleSheet.create({
 
 </SnackInline>
 
+## Preloading videos
+
+While another video is playing, a video can be loaded before being shown in the view. This allows for quicker transitions between subsequent videos and better user experience.
+
+In order to preload a video, a `VideoPlayer` with a video source has to be created. Even when the player is not connected
+to a `VideoView`, it will fill the buffers, and once it is connected to a `VideoView` it will be able to start playing without buffering.
+
+In some cases, it is beneficial to preload a video later in the screen lifecycle. In that case, a `VideoPlayer` with a
+`null` source should be created. To start preloading, the player source has to be replaced with a video source using the `replace` function.
+
+When multiple videos are preloaded, it is a good idea to limit the buffer size of the players to avoid excessive memory usage. The buffer can be increased once the player starts playing.
+
+Here is an example of how to preload a video:
+
+<SnackInline label='Video' dependencies={['expo-video']}>
+
+```jsx
+import { useVideoPlayer, VideoView, VideoSource } from 'expo-video';
+import React, { useCallback } from 'react';
```

Comment:
```suggestion
import { useState, useCallback } from 'react';
```

We can directly import `useState` and drop `React` here since we're already importing another hook.